### ISSUE TITLE
BUG: Failed to install scipy 1.7.x with pypy 3.7 in aarch64

### ISSUE THREAD

**Issue by Yikun** [scipy/scipy#15316]
### Describe your issue.

The conflicting requirements fail immediately when install scipy with pypy aarch64.

Related: https://github.com/scipy/oldest-supported-numpy/issues/39

### Reproducing Code Example

```python
Install scpi v1.7.x (latest version)

pypy3 -m pip install scipy==v1.7.3
```

There are not conlict in **master branch**, but there are still a wrong line to make pypy aarch64 to install 1.19.2

### Error message

```shell
#18 20312.1   ERROR: Cannot install numpy==1.19.2 and numpy==1.20.0 because these package versions have conflicting dependencies.
#18 20312.1   
#18 20312.1   The conflict is caused by:
#18 20312.1       The user requested numpy==1.19.2
#18 20312.1       The user requested numpy==1.20.0
```

### SciPy/NumPy/Python version information

- pypy3(3.7) aarch64
- scipy 1.7.x

**Comment by rgommers** [comment]
The fix for this is merged into the `maintenance/1.7.x` branch. If we do a `1.7.4` release, it will be fixed. Closing. Thanks @Yikun

**Comment by Yikun** [comment]
@rgommers Thanks!
